poker_workflow.py

```diff
@@ -0,0 +1,35 @@
+import ast
+from temporalio import workflow
+from game_state import GameState
+from deck_utils import deal_cards, create_deck, shuffle_deck
+from hand_ranking_workflow import HandRankingWorkflow, HandRankingInput
+
+
+@workflow.defn
+class PokerWorkflow:
+    @workflow.run
+    async def run(self, seed: int) -> None:
+        deck = await shuffle_deck(create_deck(), seed)
```

Comment:
This is resolved, but I've raised other errors which I'm attempted to resolve now....

Reply:
Hey @cretz - this will be paused for a bit while I explore a new opportunity. Thanks so much for all your help and while I may tinker with this project in the future, I don't want you to expect me to update it anytime in the near future.